aquarius/ddo_checker/shacl_schemas/v4/remote_4.2.0.ttl

```diff
@@ -0,0 +1,428 @@
+        sh:pattern "^(.*)$" ;
+        sh:minCount 1;
+        sh:maxCount 1;
+        sh:maxLength 256;
+    ] ;
+    sh:property [
+        sh:path schema:required ;
+        sh:datatype xsd:boolean ;
+        sh:minCount 1;
+        sh:maxCount 1;
+    ] ;
+    sh:property [
+        sh:path schema:description ;
+        sh:datatype xsd:string ;
+        sh:pattern "^((.|\n)*)$" ;
+        sh:minCount 1;
+        sh:maxCount 1;
+        sh:maxLength 65536;
+    ] ;
+    sh:property [
+        sh:path schema:default;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.*)$" ;
+        sh:minCount 1;
+        sh:maxCount 1;
+        sh:maxLength 256;
+    ] ;
+    sh:property [
+        sh:path schema:options;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.*)$" ;
+        sh:maxLength 256;
+    ] ;.
+
+
+schema:ContainerShape
+    sh:targetClass schema:Container ;
+    sh:property [
+        sh:path schema:entrypoint ;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.*)$" ;
+        sh:minCount 1;
+        sh:maxCount 1;
+        sh:maxLength 256;
+    ] ;
+    sh:property [
+        sh:path schema:image ;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.*)$" ;
+        sh:minCount 1;
+        sh:maxCount 1;
+        sh:maxLength 256;
+    ] ;
+    sh:property [
+        sh:path schema:tag ;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.*)$" ;
+        sh:minCount 1;
+        sh:maxCount 1;
+        sh:maxLength 256;
+    ] ;
+    sh:property [
+        sh:path schema:checksum ;
+        sh:datatype xsd:string ;
+        sh:pattern "^sha256\\:(.*)$" ;
+        sh:maxLength 71;
+        sh:minLength 71;
+        sh:minCount 1;
+        sh:maxCount 1;
+    ] ;.
+
+
+schema:ServiceShape
+    sh:targetClass schema:Service ;
+    sh:property [
+        sh:path schema:id ;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.*)$" ;
+        sh:minCount 1;
+        sh:maxCount 1;
+        sh:maxLength 256;
+    ] ;
+    sh:property [
+        sh:path schema:type ;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.*)$" ;
+        sh:minCount 1;
+        sh:maxCount 1;
+        sh:maxLength 256;
+    ] ;
+    sh:property [
+        sh:path schema:name ;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.*)$" ;
+        sh:maxCount 1;
+        sh:maxLength 256;
+    ] ;
+    sh:property [
+        sh:path schema:description ;
+        sh:datatype xsd:string ;
+        sh:pattern "^((.|\n)*)$" ;
+        sh:maxCount 1;
+        sh:maxLength 65536;
+    ] ;
+    sh:property [
+        sh:path schema:datatokenAddress ;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.*)$" ;
+        sh:minCount 1;
+        sh:maxCount 1;
+        sh:minLength 42;
+        sh:maxLength 42;
+    ] ;
+    sh:property [
+        sh:path schema:serviceEndpoint ;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.*)$" ;
+        sh:minCount 1;
+        sh:maxCount 1;
+        sh:maxLength 2048;
```

Comment:
2048 characters for the serviceEndpoint seems too long to me. I would have thought this also should have `minLength` as well?

Reply:
well, if it is a private provider, it might have a very very long service Endpoint. What would you suggest for minLength?

Reply:
I think something like `http://hi.io` is the shortest it could be, which is 12 characters. Maybe we could say 10 to leave a bit of tolerance?